### ISSUE TITLE
Current Progress/Future Ideas

### ISSUE THREAD

**Issue by clewis7** [hantman-lab/animal-soup#3]
- [ ] curate good hand labels for ground truth (roughly 1000-2000)
- [ ] train with DEG again with good hand labels 
     - [ ]  have train/test split
- [ ] build model of CNN using pytorch
- [ ] API once mega-classifier is proficient at predicting
     - [ ]  jupyter notebook using fpl for viz
     - [ ] keep ethogram cleaning (can improve on current model)
